deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,41 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+body {
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+.nav {
+    display: flex;
+    flex-basis: 20%;
+    flex-grow: 0;
+    flex-shrink: 0;
+
+}
+.div {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 15%;
+    flex-grow: 1;
+    flex-shrink: 1;
+}
+.main {
+    display: flex;
+    flex-direction: column;
+    flex-basis: auto;
+    flex-shrink: 1;
+    flex-grow: 1;
+}
+
+header, footer {
+    display: flex;
```

Comment:
Om een element te laten reageren hoef je niet perse er een extra flexbox van te maken! Nu heb je bij elke element display:flex gedaan, maar dat is onnodig!